joplin/base/views/joplin_search_views.py

```diff
@@ -94,7 +94,7 @@ def search(request):
         return render(request, "wagtailadmin/pages/search_results.html", {
             'pages': pages,
             'all_pages': all_pages,
-            'query_string': q,
+            'query_string': query,
             'content_types': content_types,
```

Comment:
The 'q' value does other things and we just want to pass the actual query,  it was causing an error when pages rendered without an actual search.